client/my-sites/hosting/sftp-card/index.js

```diff
@@ -183,7 +176,7 @@ const SftpCard = ( {
 						) }
 					</p>
 					<Button onClick={ createUser } primary>
-						{ translate( 'Enable SFTP Credentials' ) }
+						{ translate( 'Create SFTP Credentials' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Enable SFTP Credentials' )` *ES Score: 7*